src/utils/nextLevel.js

```diff
@@ -0,0 +1,26 @@
+import data from "../data/data";
+
+// expects completed to be an array of completed interactions [1,24] and returns {has:1,max:2} object
+const nextLevel = (completed) => {
+  const currentStars = completed.length;
+  let nextLevel = {};
```

Comment:
Maybe different names for a inner variable and function name? It took me a while to realise that these are two different things.

Reply:
Makes sense - removed unnecessary const and changed the function name to findNextLevel()